package-scripts/datadog-agent/prerm

```diff
@@ -11,14 +21,19 @@ if [ -f "/etc/debian_version" ] || [ "$LINUX_DISTRIBUTION" == "Debian" ] || [ "$
     else
         /etc/init.d/datadog-agent stop || true
     fi
+
+    remove_py_compiled_files
```

Comment:
We do it in the `preinst` only for rpms (i.e. RHEL-like distros).

On debian the prerm of the old package is run on upgrades and uninstalls, and the script is always run first, so everything's pretty straightforward.